### ISSUE TITLE
How to specify more than one folder path in additional arguments

### ISSUE THREAD

**Issue by milyas-salik** [dependency-check/DependencyCheck#3761]
Hi,

I am using the below extension to integrate OWASP Dependency check with Azure CI and post the results with Sonar Qube.
https://marketplace.visualstudio.com/items?itemName=InfoSupport.infosupport-owasp-dependecy-checker

My Azure CI has different projects in one solution. I want to specify all folder paths. 
But, I don't have the option to specify more than one folder path. Please check the YAML below and assist me to specify multiple scan folders.

```
steps:
- task: InfoSupport.infosupport-owasp-dependecy-checker.owasp-dependency-check.OWASPDependencyCheck@0
  displayName: 'Execute OWASP Dependency Check'
  inputs:
    scanDirectory: '"SRC/ABCSolution/XYZServices/"'
    additionalArguments: '--scan "SRC/ABCSolution/XYZAPI"'
```

When Speicify more than one paths in  as additionalArguments: '--scan "SRC/ABCSolution/XYZAPI" --scan "SRC/ABCSolution/FGHAPI"' this task fails while running the pipeline.

Please do the needfull.

**Comment by aikebah** [comment]
That's an issue you should raise with the referenced AzureCI extension authors. I would suggest you to post your issue in their Q&A section.
The CLI of DependencyCheck (which is what their extension is using in its execution) supports multiple occurrences of the --scan argument.

**Comment by milyas-salik** [comment]
Hi @aikebah , thanks for your reply!

You can see my query here it's in the latest top.
https://marketplace.visualstudio.com/items?itemName=InfoSupport.infosupport-owasp-dependecy-checker&ssr=false#qna

I already posted but have not yet received any response. I already tried with multiple scan arguments, but it fails.

**Comment by jeremylong** [comment]
On the AzureDevOps issue board for ODC I found this issue which mentions a possible solution: https://github.com/dependency-check/azuredevops/issues/84#issue-903746771

> use the Extra Arguments Field and it works with multiple exclude arguments or only once!

I'm assuming you could use the Extra Arguments field the same way with the `scan` arg. But again - no one on this board supports the AzureDevOps plugin.

**Comment by Merlijnv** [comment]
This can be done with the extension like so:
```yaml
- task: InfoSupport.infosupport-owasp-dependecy-checker.owasp-dependency-check.OWASPDependencyCheck@0
  displayName: 'Execute OWASP Dependency Check'
  inputs:
    scanDirectory: '"$(Build.SourcesDirectory)/ABCSolution/XYZServices"'
    additionalArguments: '--scan "$(Build.SourcesDirectory)/ABCSolution/XYZAPI" --scan "$(Build.SourcesDirectory)/ABCSolution/XYZAPI"'
```
